tests/queries/0_stateless/03167_improvement_table_name_too_long.sh

```diff
@@ -7,8 +7,8 @@ CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 allowed_name_length=$($CLICKHOUSE_CLIENT -mn --query="SELECT getMaxTableNameLengthForDatabase('$CLICKHOUSE_DATABASE')")
 excess_length=$((allowed_name_length + 1))  # Ensure exceeding the limit
 
-long_table_name=$(openssl rand -base64 $excess_length | tr -dc A-Za-z | head -c $excess_length)
-allowed_table_name=$(openssl rand -base64 $allowed_name_length | tr -dc A-Za-z | head -c $allowed_name_length)
+long_table_name=$(openssl rand -base64 $((3 * excess_length)) | tr -dc A-Za-z | head -c $excess_length)
```

Comment:
exactly `$excess_length` symbols might be not enough, because base64 string could contain also digits (and two other symbols apparently). `tr` will remove them and we will be left with less then `$excess_length`.